ci-jobs/scripts/start-emulator.sh

```diff
@@ -9,39 +9,18 @@ declare -r ANDROID_AVD=test
 echo "y" | $ANDROID_HOME/tools/bin/sdkmanager --install "$emulator"
 
 # Create emulator
-echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n $ANDROID_AVD -k "$emulator" --force
+echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -d "Nexus 5X" -n $ANDROID_AVD -k "$emulator" --force
 
 echo $ANDROID_HOME/emulator/emulator -list-avds
 
 echo "Starting emulator"
 
 # Start emulator in background
-nohup $ANDROID_HOME/emulator/emulator -avd $ANDROID_AVD -no-snapshot > /dev/null 2>&1 &
-
-$ANDROID_HOME/platform-tools/adb wait-for-device get-serialno
-secondsStarted=$(date +%s)
-TIMEOUT=360
-while [[ $(( $(date +%s) - secondsStarted )) -lt $TIMEOUT ]]; do
-  # Fail fast if Emulator process crashed
-  pgrep -nf avd || exit 1
-
-  processList=$(adb shell ps)
-  if [[ "$processList" =~ "com.android.systemui" ]]; then
-    echo "System UI process is running. Checking IME services availability"
-    $ANDROID_HOME/platform-tools/adb shell ime list && break
-  fi
-  sleep 5
-  secondsElapsed=$(( $(date +%s) - secondsStarted ))
-  secondsLeft=$(( TIMEOUT - secondsElapsed ))
-  echo "Waiting until emulator finishes services startup; ${secondsElapsed}s elapsed; ${secondsLeft}s left"
-done
-
-bootDuration=$(( $(date +%s) - secondsStarted ))
-if [[ $bootDuration -ge $TIMEOUT ]]; then
-  echo "Emulator has failed to fully start within ${TIMEOUT}s"
-  exit 1
-fi
+nohup $ANDROID_HOME/emulator/emulator -avd $ANDROID_AVD -accel auto -no-boot-anim -no-snapshot > /dev/null 2>&1 &
+
+# Emulator API Level 30 needs below
+${ANDROID_HOME}/platform-tools/adb wait-for-device shell 'while [[ -z $(getprop sys.boot_completed | tr -d "'"\r"'") ]]; do sleep 1; done; input keyevent 82'
```

Comment:
Codacy found an issue: [Expressions don't expand in single quotes, use double quotes for that.](https://app.codacy.com/manual/dpgraham/appium-espresso-driver/pullRequest?prid=5731265)